File: CS/WindowsApplication1/Custom DataNavigator/MyNavigatorTextViewInfo.cs
using System;
using System.Drawing;
using DevExpress.XtraEditors.ViewInfo;

namespace WindowsApplication1
{
    public class MyNavigatorTextViewInfo : NavigatorTextViewInfo
    {
        public MyNavigatorTextViewInfo(NavigatorButtonsViewInfo viewInfo)
            : base(viewInfo)
        {

        }

        protected override string GetText(int currentRecord, int count)
        {
            return String.Format("Total = {0}; Current = {1}", count.ToString("c"), currentRecord - 1);
        }
        public override Size MinSize
        {
            get
            {
                Size size = base.MinSize;
                size.Width = 180;
                return size;
            }
        }
    }
}

File: VB/WindowsApplication1/Custom DataNavigator/MyNavigatorTextViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports DevExpress.XtraEditors.ViewInfo

Namespace WindowsApplication1
	Public Class MyNavigatorTextViewInfo
		Inherits NavigatorTextViewInfo
		Public Sub New(ByVal viewInfo As NavigatorButtonsViewInfo)
			MyBase.New(viewInfo)

		End Sub

		Protected Overrides Function GetText(ByVal currentRecord As Integer, ByVal count As Integer) As String
			Return String.Format("Total = {0}; Current = {1}", count.ToString("c"), currentRecord - 1)
		End Function
		Public Overrides ReadOnly Property MinSize() As Size
			Get
				Dim size As Size = MyBase.MinSize
				size.Width = 180
				Return size
			End Get
		End Property
	End Class
End Namespace
